beacon-chain/powchain/check_transition_config.go

```diff
@@ -54,6 +55,13 @@ func (s *Service) checkTransitionConfiguration(ctx context.Context) {
 		case <-ctx.Done():
 			return
 		case <-ticker.C:
+			if s.chainStartData != nil {
+				epoch := slots.EpochsSinceGenesis(time.Unix(int64(s.chainStartData.GenesisTime), 0))
+				if epoch >= params.BeaconConfig().BellatrixForkEpoch {
```

Comment:
This doesn't seem right. We should stop post-merge (i.e ttd reaches). We should not stop post-Bellatrix fork epoch

Reply:
at the very earliest, I would think prysm should wait at least until it is finalizing post-merge.

Reply:
Thanks for the input @mrabino1. It's actually safest to stop once we have non-empty execution payloads in beacon blocks